e2e/commands/lane.e2e.1.ts

```diff
@@ -358,6 +358,27 @@ describe('bit lane command', function () {
       });
     });
   });
+  describe(`switching lanes with deleted files`, () => {
+    before(() => {
+      helper.scopeHelper.setNewLocalAndRemoteScopesHarmony();
+      helper.bitJsonc.setupDefault();
+      helper.fs.outputFile('utils/is-type.js', fixtures.isType);
```

Comment:
I know it's confusing, but these "fixtures.isType" and "fixture.isString" are for the legacy.
For harmony try using `helper.fixtures.populateComponents(1);` instead. 

Also, you probably don't need to add a real spect file and install Chai, you can have an empty file.